dotCMS/src/main/java/com/dotcms/concurrent/scheduler/DotTask.java

```diff
@@ -0,0 +1,49 @@
+package com.dotcms.concurrent.scheduler;
+
+import com.dotmarketing.util.Config;
+
+import java.io.Serializable;
+import java.time.Duration;
+
+/**
+ * DotTask is a single task (Not recurring, for it see @{@link DotRecurringTask}), the task could has or not an initial delay.
+ * The instanceId must be unique and it will be the reference next operations
+ *
+ * @author jsanca
+ * @param <T>
+ */
+public class DotTask<T extends Runnable & Serializable> implements Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)